lib/index.js

```diff
@@ -1097,6 +1101,11 @@ vmadm.stop = function vmStop(opts, callback) {
         args.push('-F');
     }
 
+    if (opts.timeout) {
+        // all args are required to be strings
+        args.push('-t', opts.timeout.toString());
+    }
+
```

Comment:
vmadm ignores the option when it's not supported. In order to tell if it can be honored here, we'd need to do yet another VM.load() which would slow down every stop action. If you think it's worth it, we can do so.